verifier/src/lib.rs

```diff
@@ -43,6 +43,7 @@ pub fn verify(
     stack_inputs: StackInputs,
     stack_outputs: StackOutputs,
     proof: ExecutionProof,
+    acceptable_opt: &AcceptableOptions,
 ) -> Result<u32, VerificationError> {
```

Comment:
I wonder if we should take a different approach here. Instead of passing `acceptable_opt` as a parameter, maybe we should just define an `OptionSet` of acceptable options and populate it with the 4 options we pre-define [here](https://github.com/0xPolygonMiden/miden-vm/blob/main/air/src/options.rs#L51) (basically 96 and 128-bit security level for recursive and non-recursive proofs). Then we pass this option set to the `verify_proof()` function below.

Reply:
Makes sense, updated!